pkg/api/validation/dynakube/oneagent.go

```diff
@@ -153,6 +155,14 @@ func imageFieldSetWithoutCSIFlag(_ context.Context, v *Validator, dk *dynakube.D
 	return ""
 }
 
+func missingCodeModulesImage(_ context.Context, v *Validator, dk *dynakube.DynaKube) string {
```

Comment:
Not that it is very important in this case but what about unit tests? Should we add one?

Reply:
Checked for the other features, we don't have any for the oneliners. + We would need to rewrite the csi.isavailable() function to be mockable, as currently it reads the env variables to see which components are enabled.